src/file-3.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:17300/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:17300/project/issues?id=sample&branch=feature4&issues=AWCR_o9X_caTItD23EDR&open=AWCR_o9X_caTItD23EDR)

[ISSUE_KEY]: AWCR_o9X_caTItD23EDR